tests/pages/__init__.py

```diff
@@ -6,7 +6,7 @@
 # "Moved Collections Abstract Base Classes to the collections.abc module."
 import collections
 
-collections.Iterable = collections.abc.Iterable
+collections.abc.Iterable = collections.abc.Iterable
```

Comment:
this seems superfluous now. Do we use `collections.Iterable` elsewhere still?

Reply:
The comment below this line is `# The above is set before importing "WebView" to avoid "AttributeError".` Maybe that's not still an issue?

Reply:
Ah, because PyPOM isn't actually compatible with Python 3.10+, so we might have to keep this one. Can a line be opted out of the hook?

Reply:
selenium stack bites again. And yeah, I didn't read that line closely enough... kinda defeats the purpose. :)

I pushed a commit to fix this by using setattr instead. Hopefully we'll be off selenium soon. :)

Reply:
Great catch btw. Code reviews ++.